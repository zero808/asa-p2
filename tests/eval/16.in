10125 1518
174 1741
4241 5295
2555 9324
367 6668
1284 10033
3133 4241
1309 4115
1889 9953
3108 8755
7795 9324
1605 8807
2832 5674
3349 4264
1284 2666
1195 2674
995 2279
3734 5941
3627 7697
84 7435
780 1010
9237 10033
3321 3795
1455 3349
1925 6623
1455 7560
2339 2832
2115 4115
868 1115
406 2451
1869 10033
1025 3734
3183 3321
367 4534
1455 4264
3445 4543
1570 4404
3183 6545
1925 3093
7560 9086
2714 10033
1234 9324
4404 9953
2555 4241
2451 8874
653 8211
5879 6668
653 3321
1869 5175
1272 4115
3316 6938
2289 3108
1712 6851
1869 9237
3795 4776
1284 6619
5674 5698
739 9318
84 780
139 2750
3627 7602
196 7560
3133 5295
5674 6668
402 3688
196 2451
5175 6619
963 5611
1805 9324
4655 4769
3872 6851
1272 6619
76 1025
447 1025
1570 2339
1349 4090
84 3133
995 8334
1284 1309
2666 8807
3025 3316
5262 6545
3108 8807
3316 4911
5262 6851
1869 4090
1284 6557
474 8778
780 6623
1713 1741
2451 2750
895 1010
5698 6102
5674 5879
2115 3316
895 7602
739 2750
6938 10033
84 3627
706 2750
3321 6545
84 5295
9086 9318
4958 8778
3734 6156
7560 9318
6156 9086
4863 6851
6668 9953
3795 7241
4769 4863
7602 7697
5295 7602
402 7241
1195 8334
1653 2339
5941 9318
6619 8807
367 1741
139 1455
1925 4241
174 6668
3821 4863
1455 5941
4090 6938
1349 1605
139 3349
4958 5674
6623 8334
4911 5175
963 4769
5262 8211
963 4776
1309 6619
963 3688
1234 7608
7331 7560
653 3872
4776 8211
1172 4863
1309 4911
3316 4090
3003 7113
76 406
3003 5076
963 3821
6545 7113
3003 4863
3316 8807
5295 7435
3025 8807
963 3795
780 1805
4769 5262
2295 6545
1476 5674
5262 7113
447 4451
895 6623
1712 5611
4863 7241
1653 1889
780 7697
402 3795
1115 1476
895 1805
3321 7113
474 2339
5295 6623
1570 5879
3025 4115
1234 3133
474 4543
1115 1653
1309 6938
5941 8874
1653 1741
4090 6557
4115 6557
780 2279
2750 6156
3321 3688
4235 7331
2750 4264
1172 3183
995 2555
1195 7602
3321 4863
1284 2714
995 9324
2055 6619
995 7608
2339 3445
739 824
3688 6545
3821 4655
995 5295
3183 3821
447 2451
2289 2666
653 3795
4543 6668
824 4264
76 739
4241 7608
76 9318
739 2451
2055 5175
2279 3093
7435 7608
780 9324
1309 2055
2666 4115
1455 6156
1570 4534
1025 2451
1115 5698
4115 10033
2674 9324
9086 10073
1010 7795
4241 7795
1713 5879
2451 3734
6156 7560
6619 8755
3093 7602
895 3133
3321 4769
4241 7602
6102 8778
1349 6938
5674 9953
402 4863
3734 9318
2289 3025
406 9086
402 4776
1349 1869
4543 5698
4404 6668
1653 4958
1805 4241
868 8778
1570 4543
3321 3821
4655 7241
6668 8778
780 8334
3025 4911
1172 5611
139 10073
4090 10033
1349 6557
1349 2115
868 9953
1605 4115
963 3003
995 1805
367 5879
76 706
2289 8807
5175 8755
196 5262
1115 2750
2555 5295
406 4235
402 3183
780 3093
2339 4958
2714 5175
76 447
3688 7113
6156 10073
3734 7560
1925 7608
2295 6851
824 4451
739 1025
2295 4769
4655 6545
367 8778
1195 3093
963 7113
2750 4451
2339 4404
3025 9237
995 3627
402 4769
402 7113
706 2451
474 1115
653 2295
1115 8778
1925 5295
3093 7697
4404 4534
1805 2674
5175 10033
3688 6851
3795 4769
653 6851
174 1476
1309 6557
139 9318
706 4235
2295 3003
3734 10073
1653 6668
2674 7697
995 2674
3093 3627
174 2339
1349 2714
739 6156
4911 6619
1869 6938
3025 4090
963 3183
447 824
824 6156
406 6156
1010 1195
1889 3445
2832 4958
1172 8211
2674 8334
8755 9237
1712 4769
474 5879
139 824
739 3349
4776 7113
2832 4404
1476 1713
895 7795
3349 10073
653 1172
367 2832
3321 4655
3003 6851
7435 8334
1805 3133
653 3003
1925 7435
3108 5175
474 5674
1713 4958
2055 10033
2666 3108
3108 6557
447 706
4911 8807
406 824
1476 3445
739 3734
1272 1349
1713 6102
995 7435
196 2750
3003 6545
824 7560
196 1455
174 4958
3872 6545
2714 4115
2451 5941
1272 4911
1272 2115
1869 4911
2289 4911
76 7331
1605 9237
4404 8778
4655 4776
5879 6102
7331 10073
1272 2289
1172 5076
3821 3872
1476 5698
3349 7331
2750 4235
1172 3321
739 9086
2055 6557
2674 4241
2115 6557
3688 5262
1653 1713
84 895
3183 7241
5698 6668
706 4451
3108 9237
76 3734
895 2674
1712 7241
4235 7560
1741 5698
2674 5295
406 5941
3093 3133
474 4958
895 1234
1234 3093
1869 2666
1172 4776
76 4235
653 5076
5175 8807
895 7697
1272 3025
739 7560
4655 5076
5611 8211
824 5941
402 4655
367 406
4241 6623
174 3445
76 196
84 7795
3821 7241
2055 8807
3795 7113
3003 3321
1234 7602
1272 6557
3627 9324
7560 8874
5611 6851
4404 5674
1476 8778
1115 4958
2289 9237
780 995
6102 6668
6851 7241
3003 4776
1234 5262
3321 3872
3734 7331
895 7435
2289 2714
3003 3795
1115 2832
402 8211
447 4264
3445 4958
474 868
7331 8874
174 367
824 8874
2295 4655
4090 9237
174 2832
474 1570
3183 5611
1284 3108
4090 5175
1712 3872
402 5076
196 4451
3321 4776
84 7608
3321 5076
84 8334
4776 6545
895 8334
1115 6102
2666 6619
1713 5698
1455 3734
4655 8211
84 7602
3688 3872
1010 3093
3734 4264
1712 4655
174 1889
2279 7697
1653 9953
4451 7560
653 5611
1712 2295
1195 2279
4241 8334
3183 7113
447 8874
780 7795
2295 5611
1010 5295
1115 4404
3734 8874
2055 4090
1476 1653
3316 9237
7697 9324
4776 4863
4769 5611
739 1455
995 1195
1349 3025
402 1172
367 1570
1010 6623
2666 4090
1284 1605
3627 4241
824 1455
2289 4090
6623 7795
84 7697
1195 6623
5611 7241
1476 4958
3321 5611
1172 3872
1010 2674
474 9953
1455 8874
868 1889
824 9318
868 5674
653 963
824 3349
2055 3025
1010 1234
3349 5941
4451 7331
2279 3133
2555 7435
8874 10073
4451 10073
1010 2555
4235 6156
1925 3133
6545 7241
1712 3183
895 1195
196 406
1653 4404
2714 8755
1309 1605
8755 8807
4235 9318
2714 3025
2832 6668
7602 9324
6623 7602
3872 4776
2295 5076
1741 2339
2295 8211
780 7608
5175 6938
3133 7602
3872 4655
2750 3734
824 2451
963 5076
1605 4911
139 6156
4090 4115
2295 7241
653 6545
2666 3316
3627 5295
3821 6851
1869 2714
7608 7697
2055 4911
1025 5941
174 1570
2714 6619
2832 6102
2339 5674
174 1653
2750 3349
706 5941
2295 4776
406 1455
780 1925
2832 5698
3688 4655
4090 6619
2714 8807
3183 6851
2666 8755
1712 3821
3349 6156
402 6851
1476 2832
4655 4863
84 2555
1476 6102
995 3133
1172 2295
76 2451
5698 5879
1172 1712
2289 5175
139 447
1272 3316
1653 5879
706 1025
739 4235
3025 3108
2339 9953
367 4543
2555 8334
653 7241
3872 7241
1712 3688
1570 6102
653 5262
3795 4655
76 7560
1284 4115
1713 2832
2115 5175
739 4264
2295 3183
4769 7113
2674 3627
3093 6623
3445 5674
3795 3872
1455 7331
780 5295
2295 4863
1869 8807
1284 2289
1869 3316
706 9318
1805 3093
2451 7331
824 9086
7608 7795
2289 3316
4264 4451
1115 5674
1309 8755
84 1195
2451 10073
4451 8874
406 2750
4115 6619
1653 6102
447 5941
1925 2674
4958 5698
4863 5076
3025 5175
2279 3627
406 9318
868 2339
868 1741
4863 6545
780 2674
4534 6668
367 9953
1115 4534
3093 4241
5295 8334
367 5698
3688 4863
6557 9237
5674 6102
2055 2714
196 3349
406 7331
2295 3795
1889 8778
1010 7608
1309 10033
1284 3316
1869 6619
1272 8755
3183 3872
1713 8778
3872 7608
3688 5611
8334 9324
2055 2666
4863 8211
1741 8778
1713 5674
1349 9237
1605 5175
1570 5674
2295 3321
4451 5941
1349 8755
2832 3445
1309 1349
3108 4911
3872 4769
995 7795
402 963
1605 4090
139 7331
3821 4776
1476 4543
139 8874
3108 6938
84 1925
5611 7113
1195 1234
4543 9953
995 1010
1272 1605
868 6668
1653 5674
3183 4863
5295 9324
3133 7795
2714 6938
2666 4911
2555 7795
2714 4911
1476 1889
3795 5076
1869 3108
5295 7697
2451 6156
402 653
76 4451
1195 5295
1869 2055
2115 3025
76 824
1605 6557
4776 7241
76 1455
2750 10073
2289 4115
4958 6102
3003 5611
1805 7608
1284 4090
2832 4534
406 706
895 3627
995 7697
447 1455
1805 7795
2115 4911
1195 3133
1010 1925
6623 7608
1925 2279
1195 9324
3133 7435
474 4404
1476 4534
84 3093
3316 10033
406 3734
3445 5698
196 4958
4115 8755
4534 5698
7113 7241
3795 6545
868 2832
3872 5262
2055 6938
1570 1741
5941 7331
2451 4264
3349 3734
963 6545
1172 4769
2714 3108
1195 1925
7241 8211
3795 4863
3688 3795
406 10073
739 10073
1889 5674
1741 4404
963 1172
706 824
4911 6557
3183 8211
1284 9237
1713 4534
1349 4911
4241 9324
367 3445
6156 7331
868 4534
1025 6156
2115 3108
3133 3627
868 1713
2451 4451
780 4241
3133 7608
1455 2750
447 9086
3872 8211
1172 3821
1234 1805
447 7560
1889 6668
2832 8778
1455 9318
1805 3627
3316 5175
2115 2666
1284 2055
4235 4264
3108 4115
5611 6545
3445 6102
367 1889
1805 7435
1010 3133
4264 8874
2339 6668
6557 8755
2451 3349
3821 7113
3627 7608
3093 7435
5674 8778
3093 9324
706 8874
2279 7435
2714 9237
447 7331
5076 5262
7435 7602
1476 4404
2289 6938
1284 5175
1570 4958
1115 4543
367 5674
4534 6102
6557 8807
406 1025
6156 9318
1195 7697
1570 6668
963 1712
895 4241
7113 8211
1172 4655
3688 5076
2055 8755
406 4451
995 7602
1889 4534
1713 4543
6623 7697
3025 6938
3108 3316
5698 9953
824 1025
474 1889
1712 5262
4769 7241
653 1712
4911 9237
4241 7697
174 1713
1653 8778
2279 9324
4769 6851
868 5698
2750 7560
1025 1455
1010 1805
1309 4090
174 868
367 474
4655 6851
653 4769
3795 5611
1234 4241
3688 3821
3795 6851
402 6545
1713 4404
1605 2666
3183 3795
1234 2279
8874 9318
1713 1889
1025 4264
1172 3688
4115 5175
3133 9324
963 2295
3025 10033
447 3734
1025 3349
402 5262
174 1115
4115 4911
8778 9953
1570 8778
2055 9237
780 7602
895 9324
2555 7608
367 1653
7435 9324
653 4776
1741 5879
1712 4776
4655 7113
4534 8778
76 4264
1349 3316
402 3872
3821 8211
3093 8334
1195 1805
780 3133
1570 2832
139 4264
76 139
4776 5262
3821 4769
1025 2750
1805 2555
4404 5879
1889 2832
2674 7608
2674 7795
2714 6557
1476 1741
6623 9324
196 739
174 474
1272 1309
1115 3445
6619 6938
4534 5674
2055 2115
402 2295
2674 3093
1570 9953
824 3734
3734 4451
196 5941
3133 7697
1713 6668
84 1805
3003 7241
780 1195
2666 9237
1805 1925
2055 2289
1115 6668
4769 5076
2674 6623
3349 4235
1712 8211
1025 9086
4264 5941
3445 4534
1605 1869
1272 4090
1025 4235
139 4451
1309 9237
995 1234
963 4655
2115 6938
4264 7331
1272 2666
1284 3025
2750 8874
139 4235
868 4543
196 706
1605 2115
1712 3003
3003 3183
3321 5262
2555 3627
1172 3003
447 9318
4769 8211
2115 6619
3821 5611
174 9953
1272 1284
5175 6557
367 1713
196 4235
1272 4958
3349 8874
4543 5879
1455 2451
447 739
895 1925
7602 7608
84 2674
1925 7795
139 196
2339 8778
4115 8807
653 7113
196 447
2289 6619
3003 3821
84 4241
4090 8807
1653 4543
2714 3316
1349 10033
4404 6102
3627 8334
1010 7697
139 706
196 9086
1712 3321
1605 3025
3133 6623
1010 4241
1115 1741
3003 4769
402 3821
4543 6102
2115 8807
1741 5674
1349 5175
6851 8211
4911 8755
2055 4115
706 6156
1455 4451
1115 1713
1025 8874
1605 3108
1925 8334
406 739
5941 6156
780 2555
706 9086
653 3183
1455 10073
1234 2674
2555 3093
1805 2279
1653 4534
868 3445
2279 7795
1272 10033
1309 2289
1605 6938
3321 8211
1653 3445
739 8874
3445 6668
4241 7435
4404 4543
1284 1349
1234 2555
402 3003
474 1713
1741 4534
9318 10073
4090 8755
2279 7602
1712 4863
7435 7697
1925 2555
3093 7608
1805 5295
4863 5611
474 1741
4534 5879
139 739
8755 10033
824 7331
4958 6668
2674 3133
4264 9086
4958 5262
1869 2289
8807 9237
1889 5879
1805 7697
3133 8334
367 1476
196 824
1869 8755
196 4264
895 3093
406 3349
7697 7795
5295 7795
76 9086
3025 8755
4451 9086
4235 5941
139 2451
2279 6623
2555 7697
174 8778
7602 7795
1115 5879
2295 5262
174 5674
2115 9237
706 7560
3183 3688
367 4958
1476 5879
2339 6102
868 5879
3821 6545
3025 6557
3025 6619
2055 3108
895 7608
1889 4543
3627 6623
3795 3821
1172 7241
1234 1925
6619 10033
3734 4235
1234 7435
824 2750
1741 1889
1272 8807
1570 3445
3872 4863
1741 6102
3316 6619
4264 6156
474 4534
8874 9086
474 1476
3445 4404
706 1455
1025 4451
2279 2555
1476 9953
995 1925
1653 5698
1115 1570
4655 5611
4863 7113
1713 9953
139 3734
4776 5076
1172 6545
3872 7113
1712 7113
3108 6619
1010 9324
4543 8778
1025 7331
4769 6545
1455 4235
196 3734
4911 6938
1925 9324
1925 7697
196 9318
1476 2339
1570 1653
2750 3872
1741 9953
2666 6557
1869 6557
3688 7241
706 3734
6623 7435
8807 10033
1309 5175
4655 5262
5698 8778
3316 4115
3321 7241
1234 8334
1605 6619
963 3872
2674 7435
7331 9086
4863 5262
6545 6851
2750 3025
4264 10073
4451 6156
474 2832
3872 5611
1272 6938
196 8874
1272 3108
174 5879
653 4863
895 995
1741 2832
1712 6545
706 7331
1284 8807
5879 9953
3795 5262
406 7560
2295 3821
1010 7435
2832 4543
1309 2115
824 4235
6557 10033
3688 4769
2279 8334
474 6102
2666 2714
3821 5076
653 3688
6545 8211
2555 7602
139 1025
84 9324
139 406
4911 10033
3349 9086
474 5698
2666 10033
1805 8334
1712 3795
402 5611
3003 4655
1309 1869
196 7331
447 10073
367 868
3321 6851
1195 2555
76 3349
1349 6619
2666 5175
1605 2714
76 5941
1284 1869
1010 7602
4776 6851
3093 5295
5879 8778
1234 5295
2339 5879
2295 3872
1272 1869
1234 7795
1284 6938
406 8874
174 6102
84 995
447 2750
3872 5076
3688 4776
868 4958
1195 7795
474 6668
2279 2674
5076 7241
1115 2339
2339 5698
1889 4404
3183 5076
1805 7602
2674 7602
868 1570
1010 2279
2750 9086
1869 2115
174 4404
1309 3025
780 3627
1272 2714
5262 7241
2295 3688
5295 7608
6938 9237
1284 4911
4115 6938
1195 3627
1570 1713
1010 3627
1570 5698
6619 9237
447 4235
1234 6623
706 739
2115 10033
1195 4241
1605 2289
868 1476
868 6102
5076 7113
84 2279
2666 3025
963 5262
895 5295
1309 2666
1349 2666
1172 7113
76 6156
4543 5674
402 3321
3108 4090
4776 5611
6557 6619
780 7435
84 1234
1309 2714
739 7331
1605 8755
3025 3872
447 6156
2750 5941
4404 4958
780 1234
7435 7795
1234 3627
895 2555
4769 4776
1234 4958
5941 10073
1172 5262
1272 5175
4958 5879
963 3321
1025 7560
4534 4958
2115 2289
824 10073
1349 2289
84 6623
7602 8334
4235 10073
3349 4451
2451 4235
139 5941
995 4241
780 8211
4264 7560
2555 3133
1925 3627
1115 3872
3445 5879
739 5941
1570 1889
6938 8807
963 4863
1309 3108
196 10073
1869 4115
1234 7697
2055 3316
5175 9237
2339 4543
474 1653
6851 7113
4264 9318
2289 8755
1309 3316
3003 3872
1889 5698
4451 9318
1889 2339
706 10073
706 4264
4235 8874
2451 9086
1284 2115
3183 4776
1010 8334
1272 2055
3316 6557
3821 5262
174 4543
4958 9953
474 3445
84 1010
6156 8874
3445 8778
739 4451
1284 8755
3183 5262
1172 3795
1115 1889
1115 9953
1869 3025
196 1025
2339 4534
4534 9953
1476 6668
406 447
1889 6102
7331 9318
2714 4090
1172 6851
2115 4090
2295 7113
1741 3445
2279 5295
3795 8211
174 5698
139 7560
1605 3316
3316 8755
2289 6557
6102 9953
4090 4911
1476 1570
6938 8755
1741 6668
367 2339
7608 8334
7560 10073
1713 2339
3688 8211
2750 7331
3003 5262
1349 8807
4543 4958
7608 9324
1309 8807
1455 9086
995 3093
1272 5262
1741 4958
1025 10073
1713 3445
3627 7795
868 1653
963 7241
5076 6545
76 10073
5076 6851
5262 5611
76 2750
3183 4655
2832 9953
3108 10033
5941 7560
4235 4451
367 4404
3093 7795
780 895
2666 6938
1653 2832
1805 6623
5076 5611
2750 9318
1925 7602
5941 9086
4534 4543
706 3349
2832 5879
3003 3688
1889 4958
3183 4769
4115 9237
1195 7608
76 8874
2279 4241
963 8211
6557 6938
1349 4115
963 6851
3349 9318
1349 3108
406 4264
2289 10033
3003 8211
1349 2055
1234 1272
196 1234
447 3349
1712 5076
2115 2714
3627 7435
2451 7560
1605 2055
995 6623
1741 4543
1195 7435
4235 9086
3445 9953
1605 10033
402 1712
1025 9318
2115 8755
3349 7560
196 6156
7795 8334
196 1272
367 6102
4404 5698
2279 7608
1272 9237
7697 8334
895 2279
5076 8211
653 3821
3734 9086
868 4404
653 4655
2451 9318
367 1115
139 9086
2555 2674
2555 6623
174 4534
3
2 6938 8755
2 3872 474
2 3133 447
