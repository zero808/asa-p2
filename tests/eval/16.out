24
7
6
